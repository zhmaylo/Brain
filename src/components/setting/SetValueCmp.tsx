import React from 'react';
import { StyleSheet, Text, TextInput, View } from "react-native";
import { WINDOW_WIDTH } from '../../constants/otherConst';

export const SetValueCmp = ({title} ) => {
    return (
        <View style={styles.itemValue}>
            <Text style={styles.titleValue}> {title}  </Text>
            <TextInput
                style={styles.inputValue}
            />
        </View>
    )
};

const styles = StyleSheet.create({
    itemValue: {
        flexDirection: 'row',
        justifyContent: 'space-between',
        alignItems: 'center',
        backgroundColor: 'white',
        padding: 5,
    },
    titleValue: {
        fontSize: 18,
        fontWeight: "600",
    },
    inputValue: {
        borderColor: "#f0f0f0",
        color: 'black',
        borderWidth: 2,
        height: 50,
        width: WINDOW_WIDTH / 1.69,
        backgroundColor: 'white',
        textAlign: 'center',
        textAlignVertical: 'center',

        fontSize: 18,
        fontWeight: "600",
    }
})