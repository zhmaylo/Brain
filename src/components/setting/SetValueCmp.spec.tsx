
import renderer from 'react-test-renderer';
import { SetValueCmp } from './SetValueCmp';


test('renders correctly', () => {
   let title =  'test title';
    const tree = renderer.create(
        SetValueCmp ({title}))
        .toJSON();
    expect(tree).toMatchSnapshot();
});
