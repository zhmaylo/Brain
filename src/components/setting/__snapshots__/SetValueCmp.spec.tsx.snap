// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flexDirection": "row",
      "justifyContent": "space-between",
      "padding": 5,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 18,
        "fontWeight": "600",
      }
    }
  >
     
    test title
      
  </Text>
  <TextInput
    allowFontScaling={true}
    rejectResponderTermination={true}
    style={
      Object {
        "backgroundColor": "white",
        "borderColor": "#f0f0f0",
        "borderWidth": 2,
        "color": "black",
        "fontSize": 18,
        "fontWeight": "600",
        "height": 50,
        "textAlign": "center",
        "textAlignVertical": "center",
        "width": 443.7869822485207,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
